.buildkite/build.rayci.yml

```diff
@@ -22,7 +22,7 @@ steps:
     tags:
       - linux_wheels
       - oss
-    instance_type: medium
+    instance_type: large
```

Comment:
per offline discussion, split into separate PR

Reply:
Open a PR: https://github.com/ray-project/ray/pull/52797. I will ping you when all CI tests pass.